lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -226,7 +204,21 @@ def __get_k8s_job_spec_template(self, ajs):
         (see pod selector) and an appropriate restart policy."""
         k8s_spec_template = {
             "metadata": {
-                "labels": {"app": self.__produce_unique_k8s_job_name(ajs.job_wrapper.get_id_tag())[:-5]}
+                "labels": {
+                    "app.kubernetes.io/name": ajs.job_wrapper.tool.tool_id,
```

Comment:
```suggestion
                    "app.kubernetes.io/name": ajs.job_wrapper.tool.id,
```

Reply:
Well, tool.tool_id is not a thing. What special characters are not allowed ? Do you maybe want to use `tool.old_id`, which is just the `id` as it appears in the tool xml ?

Reply:
ah, thanks! yea, old_id is what I was after.